tsconfig.json

```diff
@@ -4,6 +4,7 @@
   "include": [
     "src",
     "test",
+    "recommender",
```

Comment:
At some point, we probably need to convert this repo to be `lerna` managed monorepo.